xla/service/gpu/double_buffer_loop_unrolling.cc

```diff
@@ -533,9 +533,9 @@ absl::StatusOr<bool> DoubleBufferLoopUnrolling::Run(
   for (HloInstruction* while_instr : while_instrs) {
     TF_ASSIGN_OR_RETURN(WhileLoopBackendConfig config,
                         while_instr->backend_config<WhileLoopBackendConfig>());
-    if (!config.has_known_trip_count()) {
+    if (!config.has_known_trip_count() || config.known_trip_count().n() == 1) {
       VLOG(2) << while_instr->ToString()
```

Comment:
why do we need to exclude while loops with trip count of 1 from unrolling?

Reply:
since we dont have anything to unroll if the loop only has 1 iteration, we can skip the rest of the processing early.

Reply:
If the trip count is 1, we could unroll it an get rid of the loop, no? 
Also, since you said this is only for loops with a trip count of up to 8, do you think it makes sense to limit this by 8? We could make that configurable in the pass.

Reply:
Is this change here related to the rest of the PR? If yes, how? If no, can it be a separate PR?

Reply:
The change in the file is from a merge conflict when i rebased. I will fix it.
I think setting an upper limit for the unroll makes sense. I can change that.